lib/fetch.js

```diff
@@ -42,7 +30,11 @@ export function sanitizeEventAndNews(item, type) {
     strapline: item.data[`${type}.strapline`] ?
       item.data[`${type}.strapline`].value : null,
     datetime: item.data[`${type}.timestamp`] ?
-      mapDate(item.data[`${type}.timestamp`].value) : null,
+      item.data[`${type}.timestamp`].value : null,
```

Comment:
Hmmm, not sure why that is, but you can pass an array instead of a string for the value of the key path, so `[`${type}`, 'strapline', 'value']`